submissions/models.py

```diff
@@ -107,6 +109,98 @@ def from_db_value(self, value, expression, connection, context=None):
         return json.loads(value, **self.load_kwargs)
 
 
+# Has this submission been soft-deleted? This allows instructors to reset student
+# state on an item, while preserving the previous value for potential analytics use.
+DELETED = u'D'
+ACTIVE = u'A'
+STATUS_CHOICES = (
+    (DELETED, u'Deleted'),
+    (ACTIVE, u'Active'),
+)
```

Comment:
Nit: since we're moving this, let's get rid of the `u` prefix before the string literals, since we're in python 3 now and don't need 'em.